config.yaml.sample

```diff
@@ -20,7 +20,7 @@ clair:
     options:
       # PostgreSQL Connection string
       # https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNSTRING
-      source: host=localhost port=5432 user=postgres sslmode=disable statement_timeout=60000
+      source: host=clairdb port=5432 user=postgres sslmode=disable statement_timeout=60000
```

Comment:
Changing this to `host=host.docker.internal` fixed this for me on OSX without using docker-compose to set a hostname.

https://docs.docker.com/docker-for-mac/networking/